tests/attacks/test_elastic_net.py

```diff
@@ -138,6 +138,7 @@ def test_4_tensorflow_mnist(self):
                 0.0,
             ]
         )
+        print(x_test_adv[0, 14, :, 0])
```

Comment:
@beat-buesser maybe you can remove this and other print statements in the code.

Reply:
I agree.